### PR TITLE
[Method References] LPS-160434 Convert method references to field references in class extends BasePanelApp

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang ,I have fixed the bug about LegacyPortletPanelAppHotDeployListenerTest .testLegacyPortletWithControlPanelEntry.

Detial: https://github.com/liferay-core-infra/liferay-portal/pull/1786

### CODE REVIEW

**Comment by W2jx** [comment]
Hi @zhangyan0701 , Could you please help me check the test results ? Thanks!

File: modules/apps/application-list/application-list-test/src/testIntegration/resources/com/liferay/application/list/deploy/hot/test/dependencies/control-panel-entry-liferay-portlet.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <liferay-portlet-app>
 	<portlet>
-		<portlet-name>1</portlet-name>
+		<portlet-name>90</portlet-name>
```
Comment (dantewang) [inline modules/apps/application-list/application-list-test/src/testIntegration/resources/com/liferay/application/list/deploy/hot/test/dependencies/control-panel-entry-liferay-portlet.xml:5]: @W2jx Did you try to put here a custom portlet name, and in the test, deploy a test/dummy portlet with that name?

`90` is a very special portlet which means "PORTAL" itself. We should avoid using it.

**Review by dantewang — APPROVED** [review]

**Review by dantewang — CHANGES_REQUESTED** [review]
Sorry clicked the wrong option.

File: modules/apps/application-list/application-list-test/src/testIntegration/resources/com/liferay/application/list/deploy/hot/test/dependencies/control-panel-entry-liferay-portlet.xml (hunk 1 above)
Comment (W2jx) [reply modules/apps/application-list/application-list-test/src/testIntegration/resources/com/liferay/application/list/deploy/hot/test/dependencies/control-panel-entry-liferay-portlet.xml:5]: I tried to deploy a test portlet, but it didn't work. I can try again

**Pull request closed without merging**